pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: ffi
-version: 0.1.1+1
```

Comment:
I missed this in my review: We need to have `CHANGELOG.md` entries for the changes.

Reply:
Done.